docs-src/content/using-cubic-sdk/connecting.md

```diff
@@ -95,8 +95,22 @@ CubicGrpc.CubicStub mCubicService = CubicGrpc.newStub(mCubicChannel);
 ```
 {{% /tab %}}
 
-{{%/tabs %}}
+{{% tab "Swift/iOS" %}}
+``` swift
+import Cubic
+
+class CubicConnection {
+    
+    let serverAddress = "demo-cubic.cobaltspeech.com"
+    let serverPort = 2727
+
+    let client = Client(host: serverAddress, port: serverPort, useTLS: true)
```

Comment:
does "import Cubic" pull all contents into the global namespace? Or do we need to use Cubic.Client here?

Reply:
If there is no other Client class, it is enough to import Cubic and use Client without its module's name.

Reply:
:+1: